torch/nn/quantized/functional.py

```diff
@@ -227,7 +227,7 @@ def interpolate(input, size=None, scale_factor=None, mode='nearest', align_corne
                                            align_corners)
 
 def linear(input, weight, bias=None, scale=None, zero_point=None):
-    # type: (Tensor, Tensor, Optional[Tensor]) -> Tensor
+    # type: (Tensor, Tensor, Optional[Tensor], Optional[float], Optional[Tensor]) -> Tensor
```

Comment:
The last argument is `long` no? Or the doc below is wrong?

Reply:
@albanD Oh, thank you. I think type of `zero_point` should be `Optional[int]` because of line 258. Call signature of `Tensor.q_zero_point` is `def q_zero_point(self) -> _int: ...`.